docs/api.md

```diff
@@ -58,3 +58,9 @@
 .. autoclass:: demes.Admix
     :members:
 ```
+
+## Conversion functions
+
+```{eval-rst}
+.. autofunction:: demes.convert.from_ms
```

Comment:
I'm kinda anti hierarchy, unless it's really needed. ``demes.from_ms`` would be more convenient, wouldn't it? Particularly with the ``from_x`` form, I really can't see us having namespace collisions.

Reply:
Sure, that works.